java/sage/media/format/FormatParser.java

```diff
@@ -395,6 +420,38 @@ private static boolean addAdditionalMetadata(java.io.File f, ContainerFormat for
     return rv;
   }
 
+  private synchronized static FormatParserPlugin getFormatParserPluginInstance()
```

Comment:
Update this method to be:

```
private static FormatParserPlugin getFormatParserPluginInstance()
{
    /*
     * Check to see if an instance has already been created.  If not check to see if one is configured and attempt to create an instance
    */
    if (formatParserPluginInstance == null)
    {
      String parsePlugin = sage.Sage.get("mediafile_mediaformat_parser_plugin", "");

      if (parsePlugin.isEmpty())
      {
          return null;
      }
      else
      {
    synchronized (formatParserPluginLock)
    {
      if (formatParserPluginInstance != null)
        return formatParserPluginInstance;
        try        
        {
         formatParserPluginInstance = (FormatParserPlugin) Class.forName(parsePlugin, true, sage.Sage.extClassLoader).newInstance();
        } 
        catch (Throwable e1)
        {
          if (sage.Sage.DBG)
          {
              System.out.println("Error instantiating metadata parser plugin of: " + e1);
          }
        }
      }
    }
  }

    return formatParserPluginInstance;
  }

```

Reply:
@Narflex  never mind.  I see the second check for null....  Sorry